core/templates/pages/collection-player-page/collection-player-page.directive.html

```diff
@@ -221,7 +220,7 @@ <h2 ng-if="$ctrl.collectionPlaythrough.hasFinishedCollection()" class="oppia-pag
 
     <div ng-if="$ctrl.collection && $ctrl.collectionSummary" class="oppia-card-preview-panel d-none d-md-block">
       <md-card class="oppia-activity-summary-tile md-default-theme">
-        <div class="title-section" style="background-color: <[$ctrl.collectionSummary.thumbnail_bg_color]>; z-index: 1;">
+        <div class="title-section" ng-style="{'background-color': $ctrl.collectionSummary.thumbnail_bg_color, 'z-index': '1'}">
```

Comment:
Since 'z-index' is not set dynamically, I would suggest moving that rule into the definition for title-section.

Reply:
Done.